src/category_theory/monad/limits.lean

```diff
@@ -329,5 +330,23 @@ lemma has_limits_of_shape_of_reflective [has_limits_of_shape J C] (R : D ⥤ C)
 lemma has_limits_of_reflective (R : D ⥤ C) [has_limits C] [reflective R] : has_limits D :=
 { has_limits_of_shape := λ J 𝒥₁, by exactI has_limits_of_shape_of_reflective R }
 
+/--
+The reflector always preserves terminal objects. Note this in general doesn't apply to any other
+limit.
+-/
+noncomputable def reflector_preserves_terminal (R : D ⥤ C) [reflective R] [has_terminal C] :
```

Comment:
Should this be `left_adjoint_preservers_terminal_of_reflective`?

Reply:
No, it should be `left_adjoint_preserves_terminal_of_reflective` ;)